
Use DejaVu Sans as default font and adjusts sizes accordingly.
--- a/kdeui/kernel/kglobalsettings.cpp
+++ b/kdeui/kernel/kglobalsettings.cpp
@@ -374,7 +374,7 @@
 
 // NOTE: keep in sync with kdebase/workspace/kcontrol/fonts/fonts.cpp
 static const char GeneralId[] =      "General";
-static const char DefaultFont[] =    "Sans Serif";
+static const char DefaultFont[] =    "DejaVu Sans";
 #ifdef Q_WS_MAC
 static const char DefaultMacFont[] = "Lucida Grande";
 #endif
@@ -387,13 +387,13 @@
     { GeneralId, "toolBarFont", DefaultMacFont, 11, -1, QFont::SansSerif },
     { GeneralId, "menuFont",    DefaultMacFont, 13, -1, QFont::SansSerif },
 #else
-    { GeneralId, "font",        DefaultFont, 10, -1, QFont::SansSerif },
-    { GeneralId, "fixed",       "Monospace", 10, -1, QFont::TypeWriter },
+    { GeneralId, "font",        DefaultFont,  9, -1, QFont::SansSerif },
+    { GeneralId, "fixed", "DejaVu Sans Mon",  9, -1, QFont::TypeWriter },
     { GeneralId, "toolBarFont", DefaultFont,  8, -1, QFont::SansSerif },
-    { GeneralId, "menuFont",    DefaultFont, 10, -1, QFont::SansSerif },
+    { GeneralId, "menuFont",    DefaultFont,  9, -1, QFont::SansSerif },
 #endif
     { "WM",      "activeFont",           DefaultFont,  9, QFont::Bold, QFont::SansSerif },// inconsistency
-    { GeneralId, "taskbarFont",          DefaultFont, 10, -1, QFont::SansSerif },
+    { GeneralId, "taskbarFont",          DefaultFont,  9, -1, QFont::SansSerif },
     { GeneralId, "smallestReadableFont", DefaultFont,  8, -1, QFont::SansSerif }
 };
 

